app/backend/lib/s3archive.ts

```diff
@@ -95,9 +95,7 @@ s3archive.get('/api/analyst/archive', async (req, res) => {
     sortAndAppendAttachments(jsonData, ccbcId);
   });
 
-  archive.finalize();
```

Comment:
Would adding an `await` here also have solved the problem? I assume it was returning before this had finished.

Reply:
I did try that but it would still return the res.end before the archive finalized.